### DISCUSSION TITLE
Is character active contacts getter exposition missing?

### DISCUSSION THREAD

**Discussion by D8H** [jrouwe/JoltPhysics.js#226] [Q&A] [answered]
Sorry for the double post, following this first [thread](https://github.com/jrouwe/JoltPhysics.js/discussions/224), I've only realized that I may need a few other functions that seem to be missing from the IDL:
- GetActiveContacts
- HasCollidedWith

from the CharacterVirtual class.

By the way, the last version of GDevelop is using Jolt:
- https://github.com/4ian/GDevelop/releases/tag/v5.5.220

**Comment by jrouwe** [comment, marked as answer]
Should be fixed in https://github.com/jrouwe/JoltPhysics.js/commit/f235135630ae416d6b5e3a6237ba47a9a5647aa1

> By the way, the last version of GDevelop is using Jolt

Very nice! I'll add the project to the list.
